onert-micro/eval-driver/TrainingDriver.cpp

```diff
@@ -100,117 +92,236 @@ int entry(int argc, char **argv)
     throw std::runtime_error(errmsg.c_str());
   }
 
-  // Create interpreter.
-  luci_interpreter::Interpreter interpreter(model_data.data(), true);
+  return model_data;
+}
 
-  luci_interpreter::training::TrainingSettings settings;
-  settings.learning_rate = 0.0001;
-  settings.number_of_epochs = 100;
-  settings.batch_size = 1;
+bool saveModel(const char *path, const DataBuffer &data_buffer)
+{
+  if (path == nullptr or data_buffer.size() == 0)
+    return false;
 
-  const auto input_size = interpreter.getInputDataSizeByIndex(0);
-  const auto output_size = interpreter.getOutputDataSizeByIndex(0);
+  // Open or create file
+  // Note: if the file existed, it will be overwritten
+  std::ofstream out_file(path, std::ios::binary | std::ios::trunc);
+  if (not out_file.is_open())
+    return false;
 
-  char *train_data = new char[input_size * num_train_data_samples];
-  char *label_train_data = new char[output_size * num_train_data_samples];
+  // Write data
+  out_file.write(data_buffer.data(), data_buffer.size());
 
-  char *test_data = new char[input_size * num_test_data_samples];
-  char *label_test_data = new char[output_size * num_test_data_samples];
+  // Close file
+  out_file.close();
 
-  readDataFromFile(input_train_data_path, train_data, input_size * num_train_data_samples);
-  readDataFromFile(input_test_data_path, test_data, input_size * num_test_data_samples);
+  // Saving done
+  return true;
+}
```

Comment:
@chunseoklee, Introduce variant without wof file, only circle. In this variant:
We open circle file, we copy model data into RAM memory, train model (change our local model data), and then save it with current help method.

What do you think about this approach?

Reply:
Introduced  #13107 (without weight divider), please check

Reply:
> We open circle file, we copy model data into RAM memory, train model (change our local model data), and then save it with current help method.

w/o checkpoint, this approach is exactly what I have in mind. 

After resuming training from checkpoint, we load weight from checkpoint(all or some), train with them, and then save it like : copy circle into RAM and copy current weight into circle buffer and dump file.